### PR TITLE
Defined activation parameters for technical specification.

### PR DESCRIPTION
**Pull request by joshmg**

### CODE REVIEW

**Comment by BigBlockIfTrue** [comment]
Looks OK to me.

I think it would be nice if the CHIP also says a few sentences about how BCHU (CPFP) and BCHN (no CPFP) handle large chains, with the reference to Peter Tschipper's document for BCHU. Calin Culianu can give you some sentences for BCHN.

File: unconfirmed-transaction-chain-limit.md [hunk 1]
```diff
@@ -50,15 +50,8 @@ Software Verde has developed multiple applications that create and distribute tr
...
-For example, early in the CHIP's life, basic outlines of possibilities, and later in the CHIP's life, specifically architecture, etc
-https://docs.google.com/document/d/1Rgc60VipaMnbWksyXw_3L0oOqqCW2j0FKtENlqLgqYw/edit
-** Implementations **
-This section should contain code examples and proof-of-concepts of the CHIP's proposed change, or links to them. It can be updated as the CHIP makes its way through evaluation and testing.
-** Specification **
-This section should contain detailed specifications of the proposal, including but not limited to protocol formats, parameter values and expected workflows.
-** Test Cases **
-This section should contain test cases that can be used to evaluate new implementations against existing ones.
 
+The current policy limit of 50 on unconfirmed ancestors or descendants is to be removed entirely once MTP >= 1621080000 and this limit removal remains in affect even in the case of a subsequent re-org to below that MTP.
```
Comment (ftrader) [inline unconfirmed-transaction-chain-limit.md]:
```suggestion
The current policy limit of 50 unconfirmed ancestors or descendants is to be removed entirely once MTP >= 1621080000 and this limit removal remains in affect even in the case of a subsequent re-org to below that MTP.
```

I removed the superfluous 'on' which I accidentally added in my [BCR comment](https://bitcoincashresearch.org/t/chip-unconfirmed-transaction-chain-limit/302/48).

**Review by ftrader — CHANGES_REQUESTED** [review]
Just a small typo found (was my fault in the original, my wording was just bad there)

**Comment by joshmg** [comment]
> Looks OK to me.
> 
> I think it would be nice if the CHIP also says a few sentences about how BCHU (CPFP) and BCHN (no CPFP) handle large chains, with the reference to Peter Tschipper's document for BCHU. Calin Culianu can give you some sentences for BCHN.

I'll see if I can't reach them tomorrow for these additions.  Thanks, BBiT.

**Comment by joshmg** [comment]
I'm going to merge this change as-is for now and contact Calin and Stone separately from this PR to add additional content.

**Pull request merged**